schemas/iso19139/src/main/plugin/iso19139/formatter/xsl-view/view.xsl

```diff
@@ -952,4 +988,55 @@
   <xsl:template mode="render-value"
                 match="@*"/>
 
-</xsl:stylesheet>
+  <!-- isBasedOn parentidentifier -->
+
+  <xsl:template mode="render-value"
+                match="gmd:parentidentifier/gco:CharacterString">
+    <span itemprop="isBasedOn">
+      <a href="{concat($nodeUrl, 'api/records/', .)}"><xsl:value-of select="."/></a>
+    </span>
+  </xsl:template>
+
+  <!-- license -->
+  <xsl:template mode="render-value"
+                match="gmd:MD_LegalConstraints/gmd:otherConstraints/gmx:Anchor/@xlink.href">
+    <span itemprop="license"><a href="{.}"><xsl:value-of select="."/></a></span>
+  </xsl:template>
+
+  <!-- spatialCoverage	Place 
+  temporalCoverage	DateTime -->
+  <xsl:template mode="render-value"
+                match="gmd:extent/gml:TimePeriod/gml:beginPosition">
+    <span itemprop="temporalCoverage" >
+      <xsl:value-of select="."/> / 
+      <xsl:value-of select="../endPosition"/>
+    </span>
+  </xsl:template>
+
+  <!-- thumbnailUrl	URL  -->
+  <xsl:template mode="render-value"
+                match="gmd:graphicOverview/gmd:MD_BrowseGraphic/gmd:fileName/gco:CharacterString">
+    <a href="{.}" itemprop="thumbnailUrl">
+      <img src="{.}" style="max-width:250px"/>
+    </a>
+  </xsl:template>
+
+  <!-- version -->
+  <xsl:template mode="render-value"
+                match="gmd:edition/gco:CharacterString">
+    <span itemprop="version">
+      <xsl:value-of select="."/>
+    </span>
+  </xsl:template>
+
+
+  <!-- alternateName -->
+  <xsl:template mode="render-value"
+                match="gmd:alternateTitle/gco:CharacterString">
+    <span itemprop="alternateName">
+      <xsl:value-of select="."/>
```

Comment:
May be multilingual so better to match `gmd:alternateTitle` and then use `<xsl:apply-template mode="render-value" select="*"/>`. Same for edition